### PR TITLE
[DOCFIX] Improve scalability doc

### PR DESCRIPTION
**Pull request by jiacheliu3**
Add sections regarding the new worker register features, together with more thorough discussion on the master-worker sync tuning.

Also miscellaneous other small improvements across the doc.

### CODE REVIEW

File: docs/en/operation/Scalability-Tuning.md [hunk 1]
```diff
@@ -79,16 +110,16 @@ clients and then convert to a operations/second metric
 
 For example, in a deployment with 2 users, 50 Presto worker nodes (with 200 task concurrency), and
 50 Alluxio nodes, the estimations would come out to the following
-* 50 (workers) + 2 (users) x 11 (block pool size) x 10 (file pool size) x 50 (services) = 11050
+* 50 (workers) + 2 (users) x (11 (block pool size) + 10 (file pool size)) x 50 (services) = 2150
```
Comment (jiacheliu3) [inline docs/en/operation/Scalability-Tuning.md:113]: This calculation was wrong. But not sure if I'm now right.

Comment (ggezer) [reply docs/en/operation/Scalability-Tuning.md:113]: I believe the entire `Number of Concurrent Clients` section is a stretch.
I don't think we can provide meaningful numbers here that covers all possible client applications. (Concurrency setting could be executed at a single FS or multiple FSes depending on the application.)

I vote for dropping the section. @yuzhu WDYT?

File: docs/en/operation/Scalability-Tuning.md [hunk 2]
```diff
@@ -102,10 +133,11 @@ checked with the `ulimit` command.
 
 It is also important to monitor key timer metrics, as an abnormally high response rate would indicate the master is under stress.
 * Master.JournalFlushTimer 
-Journal can’t keep up with the flush, master request per second might be too high。 Consider using a more powerful master node.
+If the journal can’t keep up with the flush, master request per second might be too high. Consider using a more powerful master node.
```
Comment (ggezer) [inline docs/en/operation/Scalability-Tuning.md:136]:
```suggestion
If the journal can’t keep up with the flush, master might report request count per second that is higher than initiated by clients. Consider using a more powerful master node.
```

File: docs/en/operation/Scalability-Tuning.md [hunk 3]
```diff
@@ -102,10 +133,11 @@ checked with the `ulimit` command.
...
 
 * Master.ListStatus Timer
-Any of the RPC timer statistics would help here. If the latency is abnormally high, master is under a lot of load.  It might be time for a more powerful master node.
+Any of the RPC timer statistics would help here. If the latency is abnormally high, master is under a lot of load.
```
Comment (ggezer) [inline docs/en/operation/Scalability-Tuning.md:139]:
```suggestion
Any of the RPC timer statistics would help here. If the latency is abnormally high, master might be under a lot of load. 
```

File: docs/en/operation/Scalability-Tuning.md [hunk 4]
```diff
@@ -102,10 +133,11 @@ checked with the `ulimit` command.
 
 It is also important to monitor key timer metrics, as an abnormally high response rate would indicate the master is under stress.
 * Master.JournalFlushTimer 
-Journal can’t keep up with the flush, master request per second might be too high。 Consider using a more powerful master node.
+If the journal can’t keep up with the flush, master request per second might be too high. Consider using a more powerful master node.
 
 * Master.ListStatus Timer
-Any of the RPC timer statistics would help here. If the latency is abnormally high, master is under a lot of load.  It might be time for a more powerful master node.
+Any of the RPC timer statistics would help here. If the latency is abnormally high, master is under a lot of load. 
+It might be time for a more powerful master node.
```
Comment (ggezer) [inline docs/en/operation/Scalability-Tuning.md:140]:
```suggestion
Consider using a more powerful master node.
```

**Review by ggezer — CHANGES_REQUESTED** [review]

File: docs/en/operation/Scalability-Tuning.md (hunk 1 above)
Comment (jiacheliu3) [reply docs/en/operation/Scalability-Tuning.md:113]: +1 

I don't think this number is ever validated. Now that we have better visibility (and we can add more), we should perform some validation and add a section later which we guarantee the correctness.

Comment (yuzhu) [reply docs/en/operation/Scalability-Tuning.md:113]: yeah. i agree

**Review by yuzhu — APPROVED** [review]
LGTM, @jiacheliu3 thanks for the improvement! looks a lot better now!

**Comment by jiacheliu3** [comment]
updated the concurrency calculation sections, PTAL again @ggezer Thx!

**Review by ggezer — APPROVED** [review]

**Comment by ggezer** [comment]
alluxio-bot, merge this please

**Comment by jiacheliu3** [comment]
@Xenorith could you cherry-pick this? Thx!

**Comment by Xenorith** [comment]
alluxio-bot, cherry-pick this to branch-2.7 please

**Pull request merged**